manager/examples/clone_vs_add_hook/main.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	"github.com/sirupsen/logrus"
```

Comment:
Just noticed the use of `logrus` - how will this play with our logging in the agent?

Reply:
Oh wait, it's only used in the examples it seems - nevermind!